---
layout: default
title: Ting Wang, PhD
---
<div class="blurb">
	<a id="top"/></a>
	<h1>Welcome to my homepage !</h1>
	<div>
		<p><strong>Ting Wang</strong>, PhD</p>
		Research Associate<br>
		Fred Hutchinson Cancer Research Center<br/>
		Clinical Research Division, Tomas Building D4-255<br/>
		1100 Fairview Ave. N., Seattle, WA 98109
		<p></p>
		<strong>Email</strong>:</br>
		<ul>
			<li>wang9ting AT gmail.com</li>
			<li>twang23 AT fredhutch.org</li>
		</ul>
	</div>
	
	<div>
		<a name="education"></a>
		<h2>Education&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li><strong>2014.8 - 2018.7</strong>:<br/>
			Research Associate,<br/>
			Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA<br>
			Supervisor: Dr. <a href="https://www.fredhutch.org/en/labs/profiles/grady-william.html" style="color:blue">William Grady</a></li>
			<li><strong>2014.8 - 2018.7</strong>:<br/>
			Postdoctoral Associate,<br/>
			Dept of Pediatrics, University of Pittsburgh, Children’s Hospital of Pittsburgh of UPMC, Pittsburgh, USA<br>
			Supervisor: Dr. <a href="http://www.pitt.edu/~wec47/" style="color:blue">Wei Chen</a></li>
			<li><strong>2009.9 - 2014.7</strong>:<br/>
			Ph.D. in Control Science and Engineering, major in Bioinformatics,<br/>
			Bioinformatics Division, Center for Synthetic & Systems Biology, TNLIST, Dept. of Automation, Tsinghua University, Beijing, China<br/>
        	        Supervisor: Prof. <a href="http://www.au.tsinghua.edu.cn/publish/auen/1713/2011/20110506104703309986645/20110506104703309986645_.html" style="color:blue">Yanda Li</a></li>
			<li><strong>2005.7 - 2009.9</strong>:<br/>
			B.E. in Automatic Control,<br/>
			Dept. of Automation, Tianjin University, Tianjin, China</li>
		</ul>
	</div>
	
	<div>
		<a name="experiences"></a>
		<h2>Experiences&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>2014.8 - 2018.7: Management of computing servers in Biostatistics Lab at CHP of UPMC</li>
			<li>2009.9 - 2014.7: Secretary for the member of Chinese Academy of Sciences (Prof. Yanda Li) at THU</li>
			<li>2011.9 - 2012.1:  Teaching assistant of "Probabilistic Graphical Models" course at THU</li>
		</ul>
	</div>
	
	<div>
		<a name="researchInterests"></a>
		<h2>Research Interests&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		Interested in many topics on Bioinformatics, Biostatistics, Genetics, Network and Systems Biology, especially:<br/>
		<ul>
			<li>Integrative analysis of high-throughput and multi-omics biological data to interpret the development and progression of complex diseases, and to identify molecular biomarkers or therapeutic targets for disease diagnosis or treatment.</li>
			<li>Next generation sequencing (RNA-seq, Chip-seq, ATAC-seq, bisulfite sequencing, small-RNA sequencing, single cell sequencing) and microarray (gene expression, DNA methylation) data analysis.</li>
			<li>Systems and network analysis, pathway analysis.</li>
			<li>Design and application of algorithms in Bioinformatics.</li>
			<li>Machine learning, deep learning and application.</li>
		</ul>
	</div>
	
	<div>
		<a name="skills"></a>
		<h2>Skills&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>Theories of biostatistics, machine learning, automatic control</li>
			<li>Computing server management, pipeline design and implementation</li>
			<li>Computer languages: R, Perl, Shell, Python, C++, Matlab</li>
		</ul>
	</div>
	
	<div>
		<a name="awards"></a>
		<h2>Awards&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>Comprehensive first-class scholarship for graduate students, 2013, Tsinghua University</li>
			<li>Outstanding graduate, 2009, Tianjin University</li>
			<li>National Scholarship, 2008, Tianjin University</li>
		</ul>
	</div>
	
	<div>
		<a name="publications"></a>
		<h2>Peer-reviewed Publications&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ol>
			<li>Kristy Boggs, <strong>Ting Wang (co-first author)</strong>, Abrahim I. Orabi, Tanveer A. Javed, Tao Sun, John F. Eisses, Farzad Esni, Wei Chen, Sohail Z. Husain. Pancreatic gene expression during recovery after pancreatitis reveals unique transcriptome profiles. Scientific Reports 8.1 (2018): 1406, DOI: http://doi.org/10.1038/s41598-018-19392-0. PMID: 29362419.</li>
			<li>Tracy Tabib, Christina Morse, <strong>Ting Wang</strong>, Wei Chen, Robert Lafyatis. SFRP2/DPP4 and FMO1/LSP1 define major fibroblast populations in human skin. Journal of Investigative Dermatology (2017). DOI: https://doi.org/10.1016/j.jid.2017.09.045. PMID: 29080679.</li>
			<li>Zhe Sun, <strong>Ting Wang</strong>, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu, and Wei Chen. DIMM-SC: A Dirichlet mixture model for clustering droplet-based single cell transcriptomic data. Bioinformatics.(2017), btx490, https://doi.org/10.1093/bioinformatics/btx490. PMID: 29036318.</li>
			<ul>
				<li>Software: <strong><a href="http://www.pitt.edu/~wec47/singlecell.html" style="background-color:black; color:yellow">DIMMSC</a></strong>, an R package for clustering the droplet-based single cell RNA-seq data.</li>
			</ul>
			<li>Erick Forno, <strong>Ting Wang (co-first author)</strong>, Qi Yan, John Brehm, Edna Acosta-Perez, Angel Colon-Semidey, Maria Alvarez, Nadia Boutaoui, Michelle M. Cloutier, John F. Alcorn, Glorisa Canino, Wei Chen, Juan C. Celedón. A multi-omics approach to identify genes associated with childhood asthma risk and morbidity. American Journal of Respiratory Cell and Molecular Biology (2017). DOI: 10.1165/rcmb.2017-0002OC. PMID: 28574721.</li>
			<li>Wei Chen, <strong>Ting Wang (co-first author)</strong>, Maria Pino-Yanez, Erick Forno, Liming Liang, Qi Yan, Donglei Hu, Daniel E. Weeks, Andrea Baccarelli, Edna Acosta-Perez, Celeste Eng, Yueh-Ying Han, Nadia Boutaoui, Catherine Laprise, Gwyneth A. Davies, Julian M. Hopkin, Miriam F. Moffatt, William O.C.M. Cookson, Glorisa Canino, Esteban G. Burchard, Juan C. Celedón. An epigenome-wide association study of total serum IgE in Hispanic children. Journal of Allergy and Clinical Immunology (2017). DOI: 10.1016/j.jaci.2016.11.030. PMID: 28069425.</li>
			<li>Sandeep Puranik, <strong>Ting Wang (co-first author)</strong>, John M. Brehm, Daniel Hui, Nadia Boutaoui, Wei Chen, Erick Forno, and Juan C Celedón. Assay For Transposase-Accessible Chromatin Sequencing (ATAC-Seq) To Study The Effects Of Vitamin D On Bronchial Epithelial Cells. In A66. THE MANY FACES OF ASTHMA IN CHILDHOOD, pp. A2226-A2226. American Thoracic Society, 2017.</li>
			<li>Kong Chen, Zhe Sun, <strong>Ting Wang</strong>, Wei Chen, and Jay Kolls. Epigenetic Regulation Of Proinflammatory Chemokines In Cystic Fibrosis Lung Epithelium. In A31. MECHANISMS IN CYSTIC FIBROSIS AND OTHER BRONCHIECTATIC DISEASES, pp. A1267-A1267. American Thoracic Society, 2017.</li>
			<li><strong>Ting Wang</strong>, Richard H. Duerr, and Wei Chen. An integrative analysis of ATAC-seq and RNA-seq data in activated, CD4+ CD45RO+ CD196+ human T cells treated with IL-1B and IL-23 with or without PGE2. Proceedings of the 7th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics. ACM, 2016.</li>
			<li>Kong Chen, Taylor Eddens, Giraldina Trevejo-Nunez, Emily E. Way, Waleed Elsegeiny, David M. Ricks, Abhishek V. Garg, Carla J. Erb, Meihua Bo, <strong>Ting Wang</strong>, Wei Chen, Janet S. Lee, Sarah L. Gaffen, Jay Kolls. IL-17 Receptor Signaling in the Lung Epithelium Is Required for Mucosal Chemokine Gradients and Pulmonary Host Defense against K. pneumoniae. Cell Host & Microbe 20.5 (2016):596-605. PMID: 27923703.</li>
			<li>Kong Chen, Brian T. Campfield, Sally E. Wenzel, Jeremy P. McAleer, James L. Kreindler, Geoffrey Kurland, Radha Gopal, <strong>Ting Wang</strong>, Wei Chen, Taylor Eddens, Kathleen M. Quinn, Mike M. Myerburg, William T. Horne, Jose M. Lora, Brian K. Albrecht, Joseph M. Pilewski, Jay K. Kolls. Antiinflammatory effects of bromodomain and extraterminal domain inhibition in cystic fibrosis lung inflammation. JCI Insight 1.11 (2016): e87168. doi:10.1172/jci.insight.87168.</li>
			<li><strong>Ting Wang</strong>, Zhao Ren, Ying Ding, Zhou Fang, Matthew L. MacDonald, Robert A. Sweet, Wei Chen. FastGGM: An efficient algorithm for the inference of Gaussian graphical model in biological networks. PloS Computational Bioloy 12.2 (2016): e1004755. PMID: 26872036.</li>
			<ul>
				<li>Software: <strong><a href="http://www.pitt.edu/~wec47/FastGGM.html" style="background-color:black; color:yellow">FastGGM</a></strong>, an efficient R package for inferring Gaussian graphical model.</li>
			</ul>
			<li><strong>Ting Wang</strong>, Weihua Guan, Jerome Lin, Nadia Boutaoui, Glorisa Canino, Jianhua Luo, Juan Carlos Celedón, and Wei Chen. A systematic study of normalization methods for Infinium 450K methylation data using whole-genome bisulfite sequencing data. Epigenetics 10.7 (2015): 662-669. PMID: 26036609.</li>
			<li>Chen, Wei, John M. Brehm, Jerome Lin, <strong>Ting Wang</strong>, Erick Forno, Edna Acosta-Pérez, Nadia Boutaoui, Glorisa Canino, and Juan C. Celedón. Expression Quantitative Trait Loci (eQTL) Mapping in Puerto Rican Children. PloS one 10, no. 3 (2015). PMID: 25816334.</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Yanda Li. Inferring the perturbed microRNA regulatory networks from gene expression data using a network propagation based method. BMC bioinformatics 15.1 (2014): 255. PMID: 25069957.</li>
			<ul>
				<li>Software: <strong><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/np/" style="background-color:black; color:yellow">NP-miRNA</a></strong>, a network propagation based method for inferring perturbed miRNA regulatory networks.</li>
			</ul>
			<li>Dongfang Wang, Jin Gu, <strong>Ting Wang</strong> and Zijian Ding. OncomiRDB: a database for the experimentally verified oncogenic and tumor-suppressive microRNAs. Bioinformatics 30.15 (2014): 2237-2238. PMID: 24651967.</li>
			<ul>
				<li>Database: <strong><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/oncomirdb/" style="background-color:black; color:yellow">OncomiRDB</a></strong>, a database for the experimentally verified oncogenic and tumor-suppressive microRNAs.</li>
			</ul>
			<li><strong>Ting Wang</strong>, Jin Gu, Jun Yuan, Ran Tao, Yanda Li and Shao Li. Inferring pathway crosstalk networks using gene set co-expression signatures. Mol. BioSyst., 2013, 9(7):1822-1888. PMID: 23591523.</li>
			<ul>
				<li>Software: <strong><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/sGSCA/" style="background-color:black; color:yellow">sGSCA</a></strong>, signature-based gene set co-expression analysis using sparse canonical correlation analysis.</li>
			</ul>
		</ol>
	</div>
	
	<div>
		<a name="meetings"></a>
		<h2>Conference Presentations/Abstracts&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ol>
			<li><strong>Ting Wang</strong>, Erick Forno, Qi Yan, Nadia Boutaoui, Wei Chen and Juan C. Celedón. Epigenome- and transcriptome-wide analyses of atopic asthma in nasal epithelium and white blood cells of Hispanic children. CHP Pulmonary Research Conference, Oct. 2017. Pittsburgh, PA, USA. (Talk)</li>
			<li><strong>Ting Wang</strong>,Wei Chen and Juan C. Celedón. An epigenome-wide association study of total serum immunoglobulin E in Hispanic children. CHP Pulmonary Research Conference. Nov. 2016. Pittsburgh, PA, USA. (Talk)</li>
			<li><strong>Ting Wang</strong>, Zhao Ren and Wei Chen. FastGGM: An Efficient Algorithm for the Inference of Gaussian Graphical Model in Biological Networks. q-Med, Nov. 2016. Pittsburgh, PA, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Richard H. Duerr, Wei Chen. An integrative analysis of ATAC-seq and RNA-seq data in activated, CD4+CD45RO+CD196+ human T cells treated with IL-1B and IL-23 with or without PGE2. ACM-BCB, Oct. 2016. Seattle, WA, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Weihua Guan, Jerome Lin, Nadia Boutaoui, Glorisa Canino, Jianhua Luo, Juan Carlos Celedón, Wei Chen. A systematic study of normalization methods for Infinium 450K methylation data using whole-genome bisulfite sequencing data. ASHG, 2015. Baltimore, MD, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Zhao Ren, Ying Ding, Zhou Fang, Matthew L. MacDonald, Robert A. Sweet, Wei Chen. Construction and inference of large-scale biological networks using Gaussian graphical model. ASHG, 2015. Baltimore, MD, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Yanda Li. Network-based gene set perturbation analysis to identify causal or therapeutic miRNAs for cancers. RECOMB, 2013. Beijing, China. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jun Yuan, Ran Tao, Yanda Li, Jin Gu, Shao Li. GS-LMMA—An approach to construct the functional pathway networks for cancers. CSHL Meeting on System Biology: Global Regulation of Gene Expression. Mar. 2012. CSHL, NY, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Rui Li, Shiwen Zhao, Yanda Li. Defining gene expression adaptive responses regulated by microRNAs and transcription factors in human umbilical vein endothelial cells. CSHA Conference on Computational Biology. Oct. 2010. Suzhou, China. (Talk)</li>
		</ol>
	</div>

	<div>
		<a name="editorial"></a>
		<h2>Editorial Experiences&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ol>
			<li><strong>Editorial Board</strong>: Case Reports Journal; Annals of Medical Case Reports; SM Journal of Chronic Diseases; SM Pulmonology & Clinical Research; Immunology & Auto immune Diseases; Chronic Journal of Allergy and Immunology.</li>
			<li><strong>Invited Reviewer</strong>: Bioinformatics; Scientific Reports; PLOS ONE; BMC Bioinformatics; Frontiers of Computer Science; Pediatric Allergy, Immunology, and Pulmonology; SAGE Open Medicine; BioMed Research International; Infectious Diseases; SM Journal of Chronic Diseases; Journal of the Royal Society Interface; Mediators of Inflammation; International Journal of Carcinogenesis and Mutagenesis.</li>
		</ol>
	</div>
</div><!-- /.blurb -->
