---
layout: default
title: Ting Wang, PhD
---
<div class="blurb">
	<a id="top"/></a>
	<h1>Ting Wang, PhD</h1>
	<div>
		<p>Postdoctoral Associate</p>
		Division of Pulmonary Medicine, Allergy and Immunology, Department of Pediatrics,<br/>
		University of Pittsburgh, Children’s Hospital of Pittsburgh of UPMC<br/>
		4401 Penn Ave., Rangos Research Building, 9th floor<br/>
		Pittsburgh, PA 15224
		<p></p>
		<strong>Email</strong>:</br>
		<ul>
			<li>wang9ting AT gmail.com</li>
			<li>ting.wang5 AT chp.edu</li>
			<li>tiw33 AT pitt.edue</li>
		</ul>
	</div>
	
	<div>
		<a name="education"></a>
		<h2>Education&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li><strong>2014.8 - present</strong>:<br/>
			Postdoctoral Associate,<br/>
			Dept of Pediatrics, University of Pittsburgh, Children’s Hospital of Pittsburgh of UPMC, Pittsburgh, USA<br>
			PI: Dr. <font color="blue"><a href="http://www.pitt.edu/~wec47/">Wei Chen</a></font></li>
			<li><strong>2009.9 - 2014.7</strong>:<br/>
			PhD Student of Bioinformatics,<br/>
			Dept. of Automation, Tsinghua University, Beijing, China<br/>
        	        Supervisor: Prof. <font color="blue"><a href="http://www.au.tsinghua.edu.cn/publish/auen/1713/2011/20110506104703309986645/20110506104703309986645_.html">Yanda Li</a></font></li>
			<li><strong>2005.7 - 2009.9</strong>:<br/>
			Undergraduate Student of Automatic Control,<br/>
			Dept. of Automation, Tianjin University, Tianjin, China</li>
		</ul>
	</div>
	
	<div>
		<a name="experiences"></a>
		<h2>Experiences&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>2014.8 - present: Management of computing server in Dr. Wei Chen’s Lab</li>
			<li>2009.9 - 2014.7: Secretary for the member of Chinese Academy of Sciences (Prof. Yanda Li)</li>
			<li>2011.9 - 2012.1:  Teaching assistant of "Probabilistic Graphical Models" course</li>
		</ul>
	</div>
	
	<div>
		<a name="researchInterests"></a>
		<h2>Research Interests&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		Interested in many topics on Bioinformatics, Biostatistics, Genetics, Network and Systems Biology, especially:<br/>
		<ul>
			<li>Integrative analysis of high-throughput and multi-omics biological data to interpret the development and progression of complex diseases, and to identify molecular biomarkers or therapeutic targets for disease diagnosis or treatment</li>
			<li>Design and application of algorithms in Bioinformatics</li>
		</ul>
	</div>
	
	<div>
		<a name="skills"></a>
		<h2>Skills&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>Microarray (gene expression, DNA methylation) and deep sequencing data (RNA-seq, Chip-seq, ATAC-seq) analysis</li>
			<li>Systems and network analysis</li>
			<li>Theories of biostatistics, machine learning, automatic control</li>
			<li>Computer languages: R, Perl, C++, Matlab</li>
		</ul>
	</div>
	
	<div>
		<a name="awards"></a>
		<h2>Awards&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ul>
			<li>Comprehensive first-class scholarship for graduate students, 2013, Tsinghua University</li>
			<li>Outstanding graduate, 2009, Tianjin University</li>
			<li>National Scholarship, 2008, Tianjin University</li>
		</ul>
	</div>
	
	<div>
		<a name="publications"></a>
		<h2>Publications&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ol>
			<li><strong>Ting Wang</strong>, Zhao Ren, Ying Ding, Zhou Fang, Matthew L. MacDonald, Robert A. Sweet, Wei Chen. FastGGM: An efficient algorithm for the inference of Gaussian graphical model in biological networks. (Under Review)</li>
			<ul>
				<li>Software: <strong><font color="red"><a href="http://www.pitt.edu/~wec47/FastGGM.html">FastGGM</a></font></strong>, an efficient R package for inferring Gaussian graphical model.</li>
			</ul>
			<li><strong>Ting Wang</strong>, Weihua Guan, Jerome Lin, Nadia Boutaoui, Glorisa Canino, Jianhua Luo, Juan Carlos Celedón, and Wei Chen. A systematic study of normalization methods for Infinium 450K methylation data using whole-genome bisulfite sequencing data. Epigenetics 10.7 (2015): 662-669. PMID: 26036609.</li>
			<li>Chen, Wei, John M. Brehm, Jerome Lin, <strong>Ting Wang</strong>, Erick Forno, Edna Acosta-Pérez, Nadia Boutaoui, Glorisa Canino, and Juan C. Celedón. "Expression Quantitative Trait Loci (eQTL) Mapping in Puerto Rican Children." PloS one 10, no. 3 (2015). PMID: 25816334.</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Yanda Li. Inferring the perturbed microRNA regulatory networks from gene expression data using a network propagation based method. BMC bioinformatics 15.1 (2014): 255. PMID: 25069957.</li>
			<ul>
				<li>Software: <strong><font color="red"><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/np/">NP-miRNA</a></font></strong>, a network propagation based method for inferring perturbed miRNA regulatory networks.</li>
			</ul>
			<li>Dongfang Wang, Jin Gu, <strong>Ting Wang</strong> and Zijian Ding. OncomiRDB: a database for the experimentally verified oncogenic and tumor-suppressive microRNAs. Bioinformatics 30.15 (2014): 2237-2238. PMID: 24651967.</li>
			<ul>
				<li>Database: <strong><font color="red"><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/oncomirdb/">OncomiRDB</a></font></strong>, a database for the experimentally verified oncogenic and tumor-suppressive microRNAs.</li>
			</ul>
			<li><strong>Ting Wang</strong>, Jin Gu, Jun Yuan, Ran Tao, Yanda Li and Shao Li. Inferring pathway crosstalk networks using gene set co-expression signatures. Mol. BioSyst., 2013, 9(7):1822-1888. PMID: 23591523.</li>
			<ul>
				<li>Software: <strong><font color="red"><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/sGSCA/">sGSCA</a></font></strong>, signature-based gene set co-expression analysis using sparse canonical correlation analysis.</li>
			</ul>
		</ol>
	</div>
	
	<div>
		<a name="meetings"></a>
		<h2>Meetings&nbsp;&nbsp;<a href="#top" >Top</a></h2>
		<ol>
			<li><strong>Ting Wang</strong>, Weihua Guan, Jerome Lin, Nadia Boutaoui, Glorisa Canino, Jianhua Luo, Juan Carlos Celedón, Wei Chen. A systematic study of normalization methods for Infinium 450K methylation data using whole-genome bisulfite sequencing data. ASHG, 2015. Baltimore, MD, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Zhao Ren, Ying Ding, Zhou Fang, Matthew L. MacDonald, Robert A. Sweet, Wei Chen. Construction and inference of large-scale biological networks using Gaussian graphical model. ASHG, 2015. Baltimore, MD, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Yanda Li. Network-based gene set perturbation analysis to identify causal or therapeutic miRNAs for cancers. RECOMB, 2013. Beijing, China. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jun Yuan, Ran Tao, Yanda Li, Jin Gu, Shao Li. GS-LMMA—An approach to construct the functional pathway networks for cancers. CSHL Meeting on System Biology: Global Regulation of Gene Expression. Mar. 2012. CSHL, NY, USA. (Poster)</li>
			<li><strong>Ting Wang</strong>, Jin Gu, Rui Li, Shiwen Zhao, Yanda Li. Defining gene expression adaptive responses regulated by microRNAs and transcription factors in human umbilical vein endothelial cells. CSHA Conference on Computational Biology. Oct. 2010. Suzhou, China. (Oral presentation)</li>
		</ol>
	</div>
</div><!-- /.blurb -->
